### ISSUE TITLE
关于yolo_loss官方文档的 stop_gradient=True

### ISSUE THREAD

**Issue by 670133189** [PaddlePaddle/Paddle#37015]
官方文档：https://www.paddlepaddle.org.cn/tutorials/projectdetail/2412195
`label_objectness.stop_gradient=True
label_location.stop_gradient=True
label_classification.stop_gradient=True
scales.stop_gradient=True`
请教为何要设置 stop_gradient=True ？这样得到的 loss 就无法反向传播计算梯度：loss.backward()
